src/Resources/Locales/en.json

```diff
@@ -1148,6 +1148,14 @@
         "toggleSelection": "Toggle to Select Twin",
         "selectRow": "Twin Selected",
         "inspectProperty": "Inspect Property",
-        "results": "Results ({{twinCount}})"
+        "results": "Results ({{twinCount}})",
```

Comment:
we are trying to keep the key names reflecting where the value is used rather than the actual text it represents. ex: `ResultsListTitle`, `addRowText`, `AndOrColumnHeader`